Formula/kontena-cli.rb

```diff
@@ -0,0 +1,185 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container and microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git",
+      :tag => "v1.3.4",
+      :revision => "3588a33dfc992a0fb29603002ac33bfbe047c673",
+      :branch => "v1.3"
+
+  head "https://github.com/kontena/kontena.git"
+
+  depends_on :ruby => "2.1"
+
+  resource "addressable" do
```

Comment:
the resources are only relevant for stable so they belong in a `stable do` if you want to have a head spec.